packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js

```diff
@@ -50,17 +50,25 @@ const SettingsNav = ({ menu }) => {
       <SubNavSections>
         {sections.map((section) => (
           <SubNavSection key={section.id} label={formatMessage(section.intlLabel)}>
-            {section.links.map((link) => (
-              <SubNavLink
-                as={NavLink}
-                withBullet={link.hasNotification}
-                to={link.to}
-                onClick={() => handleClickOnLink(link.to)}
-                key={link.id}
-              >
-                {formatMessage(link.intlLabel)}
-              </SubNavLink>
-            ))}
+            {section.links.map((link) => {
```

Comment:
I'm wondering if it would be better if the `useSettingsMenu` hook would filter items out, that the user doesn't have have permissions to see (basically `.filter`ing the global and admin links based on `isDisplayed`).

What do you think?

Reply:
Oh! why didn't i thought of it